digiwf-integrations/digiwf-s3-integration/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>digiwf-s3-integration</artifactId>
+    <version>0.1.1</version>
+    <name>digiwf-s3-integration</name>
+    <packaging>pom</packaging>
+    <description>S3 integration used by DigiWF</description>
+
+    <parent>
+        <groupId>io.muenchendigital.digiwf</groupId>
+        <artifactId>digiwf-integrations</artifactId>
+        <version>0.12.0-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <minio.version>8.3.9</minio.version>
+        <digiwf.spring.cloudstream.utils.version>0.10.0</digiwf.spring.cloudstream.utils.version>
```

Comment:
The spring cloud stream utils are already migrated to the monorepo. You can remove the version here.